### PR TITLE
Schema: make notification_history#text longtext

### PR DESCRIPTION
**Pull request by Al2Klimov**

### CODE REVIEW

**Comment by Al2Klimov** [comment]
## Blocked by

* https://github.com/Icinga/icingadb/pull/421

File: schema/mysql/upgrades/1.0.0.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE notification_history
+    MODIFY `text` longtext NOT NULL;
```
Comment (julianbrost) [inline schema/mysql/upgrades/1.0.0.sql:2]: This file should also insert a new row into `icingadb_schema` and update the version check in the code.

Not yet relevant for this PR but for the next one touching the schama: we have to decide if we want to do a version bump per release or per change in the development phase, i.e. should we increase the version with each PR that updates the schema?

**Review by julianbrost — CHANGES_REQUESTED** [review]

File: schema/mysql/upgrades/1.0.0.sql (hunk 1 above)
Comment (Al2Klimov) [reply schema/mysql/upgrades/1.0.0.sql:2]: +1 per release (look at IDO)

Comment (lippserd) [reply schema/mysql/upgrades/1.0.0.sql:2]: I would see the `1.0.0.sql` as the baseline and then start with incremental schema upgrades. Conceptually, something like https://flywaydb.org/documentation/concepts/migrations#sql-based-migrations.

**Comment by lippserd** [comment]
Please rebase.

**Review by lippserd — CHANGES_REQUESTED** [review]
The version increment is missing in `schema/mysql/schema.sql`.

**Review by lippserd — APPROVED** [review]

**Comment by Al2Klimov** [comment]
PING @julianbrost

**Review by julianbrost — APPROVED** [review]

**Pull request merged**